hca/regenerate_api.py

```diff
@@ -320,39 +359,36 @@ def generate_python_bindings(test_api_path=None):
 
     # Blow away previous files in the autogenerated folder and remake that directory
     dirname = os.path.dirname(__file__)
+    api_function_path = os.path.join(dirname, "api", "__init__.py")
+    if os.path.exists(api_function_path):
+        os.remove(api_function_path)
```

Comment:
Truncate and ftruncate don't support Windows until v3.5. Given that we're focused on providing cross-platform support, sticking with these more base methods might be better. Thoughts?